codalab/client/local_bundle_client.py

```diff
@@ -474,7 +474,7 @@ def cat_target(self, target, out):
     # Maximum number of bytes to read per line requested
     MAX_BYTES_PER_LINE = 128
 
-    def head_target(self, target, max_num_lines):
+    def head_target(self, target, max_num_lines, sanitize=False):
```

Comment:
Not sure 'sanitize' the right variable here.
From this code, it's not clear that what is going on is that '<binary>' is returned if it's a binary file and otherwise, we return the original string.